<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.482955 rgba(117, 66, 201, 255), stop:0.982955 rgba(0, 85, 255, 255), stop:1 rgba(255, 255, 255, 255));

font-family: Arial, verdana;
}

QLineEdit{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.176136 rgba(212, 212, 212, 255), stop:0.835227 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
borde: 0;
font-size: 12px;
height: 25px;

}

QLabel{
color: rgb(255, 255, 255);
font-size: 12px;
}


QPushButton#BotonA{
background-color:rgb(75, 75, 75);
font-size: 12px;
color: rgb(255, 255, 255);
borde: 0;
}

QPushButton#BotonA:hover{
background-color:rgb(255, 255, 255);
color:rgb(40, 40, 40);
font-size: 12px;
}

QSpinBox{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.176136 rgba(212, 212, 212, 255), stop:0.835227 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
font-size: 12px;
height: 25px;

}


</string>
  </property>
  <widget class="QPushButton" name="BotonA">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Validar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>91</y>
     <width>200</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NOMBRE:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="NombreA">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>APELLIDO:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ApellidoA"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>EMAIL:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EmailA"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>EDAD:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="EdadA">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
